core/src/main/scala/tech/beshu/ror/accesscontrol/blocks/variables/runtime/RuntimeResolvableVariable.scala

```diff
@@ -224,4 +238,15 @@ object MultiExtractable {
         .map(NonEmptyList.one)
     }
   }
+
+  case object AvailableGroupsVar extends MultiExtractable with VariableType.AvailableGroups {
+    private val singleAvailableGroupsExtractable = SingleExtractable.AvailableGroupsVar
+
+    override def extractUsing(requestContext: RequestContext,
+                              blockContext: BlockContext): Either[ExtractError, NonEmptyList[String]] = {
+      singleAvailableGroupsExtractable
```

Comment:
I think this implementation is not correct. Please check out our docs: https://github.com/beshu-tech/readonlyrest-docs/blob/master/elasticsearch.md#dynamic-variables-from-jwt-claims.

This is used for `exploding` a variable. Here, available groups are always list, so we don't have to parse string to sprit it (like we do eg. with `JwtPayloadVar`).

Could you please reimplement it?

Reply:
Yes, you right. I didn't know there is the implementation for lists. I suppose it has to be `csv`. I will check it.

Reply:
you know, `MultiExtractable` trait was created to explode single value into list of values. Here you have a list, so in implementation of `MultiExtractable` all you need to do is just convert the list of groups to list of strings. In contrast to `SingleExtractable` where you had to merge the list to one string.

Reply:
Ok, I changed it (I moved evaluation to `MultiExtractable`) but I cannot remove `AvailableGroupsVar` from `SingleExtractable` because of it depends on https://github.com/sscarduzio/elasticsearch-readonlyrest-plugin/pull/566/files#diff-7bfb77d84aae07205a914b05b3b3886aR174 and it depends on https://github.com/sscarduzio/elasticsearch-readonlyrest-plugin/pull/566/files#diff-7bfb77d84aae07205a914b05b3b3886aR163. So I created exception there.